src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/Orchestrators/Mappers/CommitmentMapperTests/WhenMappingToTransferCommitmentViewModel.cs

```diff
@@ -109,6 +112,9 @@ public void ThenMappingACommitmentWithAppenticesMapsFieldsCorrectly(TransferAppr
             _commitmentView.TransferSender.TransferApprovalStatus = status;
             var result = _sut.MapToTransferCommitmentViewModel(_commitmentView);
             Assert.AreEqual($"XYZ{_commitmentView.TransferSender.Id}", result.HashedTransferSenderAccountId);
+            Assert.AreEqual($"CDE{_commitmentView.TransferSender.Id}", result.PublicHashedTransferSenderAccountId);
```

Comment:
My usual bugbear :-)
By using _commitmentView properties in the expected values, which the sut can mutate, can cause the test to give false results.

Reply:
done